Resources/Prototypes/_RMC14/Roles/Ranks/UNMC/enlisted.yml

```
@@ -13,7 +13,7 @@
 - type: rank
   id: RMCRankFirstSergeant
   name: First Sergeant
-  prefix: 1Sgt.
+  prefix: 1st Sgt.
```

```suggestion
  prefix: 1stSgt.
```
To keep the character count lower but , full character names will already be taking up a lot of space

---

Sure thing!